FormalConjectures/ForMathlib/Combinatorics/SimpleGraph/GraphConjectures/Invariants.lean

```diff
@@ -20,16 +20,25 @@ import Mathlib.Combinatorics.SimpleGraph.Metric
 import Mathlib.Data.Multiset.Sort
 import Mathlib.Order.CompletePartialOrder
 
+import Mathlib.Data.Real.Basic
+import Mathlib
```

Comment:
Could you run `#min_imports` at the bottom of the file? Ideally we wouldn't import all of Mathlib in this file